Source/JavaScriptCore/wasm/WasmFunctionIPIntMetadataGenerator.cpp

```diff
@@ -126,6 +126,9 @@ void FunctionIPIntMetadataGenerator::addLEB128V128Constant(v128_t value, size_t
 
 void FunctionIPIntMetadataGenerator::addReturnData(const FunctionSignature& sig)
 {
+    auto gprToIndex = [&](GPRReg r) -> unsigned {
+        return GPRInfo::toArgumentIndex(r);
+    };
```

Comment:
I think just using `GPRInfo::toArgumentIndex` instead of having `gprToIndex` lambda is better.